src/main/java/phonebook/Serialization.java

```diff
@@ -0,0 +1,33 @@
+package phonebook;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+public class Serialization {
+
+    public static void serialize(Object obj, File file) throws IOException {
```

Comment:
Cool. During lesson we said that we may have all in memory. You did more than required :)